src/App.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import Login from "./pages/Login";
+import "./App.css";
+
+function App() {
+  const handleLogin = (username: string, password: string) => {
+    console.log("Login details:", username, password);
+  };
+
+  return (
+    <div className="text-center">
+      <Login onLogin = {handleLogin} />
```

Comment:
This login component will become a page. so it is better to avoid passing props for it.